### ISSUE TITLE
Couldn't find all answers

### ISSUE THREAD

**Issue by naserahmadi** [google-research/tapas#137]
Hello,
I am trying to fine-tune tapas on my table following the steps  [here](https://huggingface.co/transformers/model_doc/tapas.html#tapasforsequenceclassification). I generated my dataset by adding coordinators and answer_text as follows:

![image](https://user-images.githubusercontent.com/45039751/132009441-fc5ec993-e971-457e-b67a-d103005cdb90.png)

but when I'm running `TableDataset`, an error appears: `ValueError: Couldn't find all answers`. Can you help me with this error? How can i fix it?

**Comment by AyKhMoEn** [comment]
Hi,

I am running into the exact same issue, would you mind sharing how you resolved it ?

Thanks,

**Comment by keyvez** [comment]
+1 Me too

**Comment by CallumJWilson** [comment]
I also had this issue. It seems the table I was using was too large (163 rows x 16 columns) and perhaps as a result when truncating the table some of the answer values were removed.

To solve this I just made the table smaller (20 rows x 7 columns) and updated my questions and answer locations accordingly. I am unsure if reducing both the number of columns and rows is necessary, or what the maximum possible table size is. Perhaps someone else can provide more insight here?
